README.md

```diff
@@ -58,6 +58,40 @@ long_line_under_100_chars
 # Extra Specs Coding Style
 
 - Add a blank line between `let` and `before`.
+- Group specs around the described object's public methods. Use `describe "#method_name` for instance methods and `describe ".method_name"` for class methods.
+- Context setup does not belong to the `it` block, that's the place for the assert.
+- Context description should always start like `context "when ..."` or `context "and ..."`
+- When you see `it "returns foo when baz is true"` it means you're missing a `context "when baz is true"`
```

Comment:
Already referred here: http://www.betterspecs.org/#contexts

Reply:
True, but I guess it doesn't hurt to emphasize 😄 Shall I remove this?

Reply:
When there is too much to read, people tend to read less.
That's why I would remove it.